strongbox-resources/strongbox-common-resources/src/main/resources/logback-debug.xml

```diff
@@ -7,6 +7,8 @@
         <resetJUL>true</resetJUL>
     </contextListener>
 
+    <springProperty scope="context" name="loggingPath" source="strongbox.vault" defaultValue="."/>
```

Comment:
https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-logback-extensions

> Because the standard logback.xml configuration file is loaded too early, you cannot use extensions in it. You need to either use logback-spring.xml